kak-lsp.toml

```diff
@@ -180,6 +180,8 @@ args = ["-c", "if command -v rustup >/dev/null; then $(rustup which rls); else r
 # roots = ["Cargo.toml"]
 # command = "sh"
 # args = ["-c", "if command -v rustup >/dev/null; then $(rustup which rust-analyzer); else rust-analyzer; fi"]
+# [language.rust.initialization_options]
+# diagnostics.disabled = ["unresolved-proc-macro"]
```

Comment:
I'm fine with merging this unless you think we should instead show how to enable proc macro support.
There are [some reasons](https://github.com/rust-analyzer/rust-analyzer/issues/6448) why it's not the default, so just disabling the warning seems like the safer choice.